dashboard/app/controllers/levels_controller.rb

```diff
@@ -45,7 +45,7 @@ def edit
   # GET all the information for the mini rubric
   def get_rubric
     @level = Level.find_by(id: params[:level_id])
-    if @level.mini_rubric.to_bool
+    if @level.mini_rubric && @level.mini_rubric.to_bool
```

Comment:
Alternative: The safe-navigation operator
```suggestion
    if @level.mini_rubric&.to_bool
```

Reply:
cool! thanks for sharing! I will update